src/sqlite/sqlite3.c

```diff
@@ -42867,8 +42869,10 @@ SQLITE_API int sqlite3_os_init(void){
 
   /* Register all VFSes defined in the aVfs[] array */
   for(i=0; i<(sizeof(aVfs)/sizeof(sqlite3_vfs)); i++){
-    sqlite3_vfs_register(&aVfs[i], i==0);
+    // We DON'T register any other vfs, so duckdbvfs is the only one used
+    //sqlite3_vfs_register(&aVfs[i], i==0);
   }
+  sqlite3_vfs_register(sqlite3_duckdbvfs(), 0);
```

Comment:
I wonder if there's a way of doing this without adjusting the SQLite files? We might update these at some point.